perfmetrics/scripts/continuous_test/ml_tests/tf/readme.txt

```diff
@@ -0,0 +1,15 @@
+This file contains the code flow for tf based kokoro job.
```

Comment:
please rename it to README.md and use the readme file format for the write up:https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax